pkg/operator/ceph/file/dependent.go

```diff
@@ -49,3 +70,43 @@ func CephFilesystemDependents(clusterdCtx *clusterd.Context, clusterInfo *client
 
 	return deps, nil
 }
+
+// return subvolume groups that have 1 or more subvolumes present in them
+func subvolumeGroupDependents(clusterdCtx *clusterd.Context, clusterInfo *client.ClusterInfo, filesystem *v1.CephFilesystem) (*dependents.DependentList, error) {
+	baseErr := "failed to get Ceph subvolume groups containing subvolumes"
+
+	deps := dependents.NewDependentList()
+
+	svgs, err := client.ListSubvolumeGroups(clusterdCtx, clusterInfo, filesystem.Name)
+	if err != nil {
+		return deps, errors.Wrap(err, baseErr)
+	}
+
+	errs := []error{}
+	for _, svg := range svgs {
+		svs, err := client.ListSubvolumesInGroup(clusterdCtx, clusterInfo, filesystem.Name, svg.Name)
+		if err != nil {
+			errs = append(errs, errors.Wrapf(err, "failed to list subvolumes in subvolume group %q", svg.Name))
+		}
+
+		if svg.Name == indexSubvolumeGroupName {
```

Comment:
can we move this check to line 87? we don't need to list subvolumes in these subvolumegroups?